packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 51.10.0 - April 21, 2021
+
+## Fixed
+
+#### docs
+
+- Update docs to reflect Apex library setting has been removed ([PR #3148](https://github.com/forcedotcom/salesforcedx-vscode/pull/3148))
+
+#### salesforcedx-vscode-apex
+
+- Update "All tests" option for `SFDX: Run Apex Tests` command to match CLI behavior ([PR #3126](https://github.com/forcedotcom/salesforcedx-vscode/pull/3126))
```

Comment:
Good catch - the change log generation didn't pick this one up correct?

Reply:
It actually did I just reworded it :)